components/EntityItem.vue

```diff
@@ -13,10 +16,27 @@
         :ui="{ base: 'pr-4 rounded-full' }"
       />
     </span>
+    <div
+      v-else
+      ref="lazyLoadTrigger"
+      class="flex items-center gap-2"
+    >
+      <UAvatar
+        :alt="name"
+        size="xs"
+        :src="avatar"
+      />
+      <span
+        class="font-semibold text-gray-900 truncate max-w-[160px]"
```

Comment:
Please use Nuxt UI text color class instead of hardcoded color class

ref: https://ui3.nuxt.com/getting-started/theme#neutral